onecodex/models/sample.py

```diff
@@ -19,11 +19,43 @@ def __repr__(self):
 
     @classmethod
     def where(cls, *filters, **keyword_filters):
+        """Query and retrieve a set of samples.
+
+        Parameters
+        ----------
+        limit : `int`, optional
+            If set, retrieve a maximum of `limit` samples. Note a limit of 1000 samples is automatically
+            enforced for queries of all public samples (`public=True`)
+        organization : `bool`, optional
+            If True, search all samples within your organization (including your samples)
+        public : `bool`, optional
+            If True, search all public samples (limited to 1000 results)
```

Comment:
it should mention `organization` and `public` are exclusive of each other, you can't look for samples either public *or* within the org